tutorials/shells/cygwin.md

```diff
@@ -1,50 +1,61 @@
 
-## How do I get it?
-1. Go to the [Cygwin homepage](http://cygwin.org), scroll down, and download the latest `setup.exe` appropriate for your architecture (most should choose 64-bit).
-2. Run the installer that you just downloaded. Accept the User Account Control dialog if it appears.
-3. Click "Next", choose to install from the internet, and click "Next" twice to accept the root install and local package directories. Click "Next" to accept your internet connection settings.
-4. Select any package download site from the list. Click "Next" to download the packages from the mirror download site.
-5. At this screen, you can search and select packages to install. I recommend `curl`, `wget`, `openssh`, `git`, and `vim` to get started. You can expand package categories using the `+` to the left of the category name. 
-![Package List](https://raw.githubusercontent.com/jay-hankins/CSToolsCourse/master/tutorials/shells/cygwin_screenshots/Packages.PNG)
+How do I get it?
+----------------
+
+1.	Go to the [Cygwin homepage](http://cygwin.org), scroll down, and download the latest `setup.exe` appropriate for your architecture (most should choose 64-bit).
+2.	Run the installer that you just downloaded. Accept the User Account Control dialog if it appears.
+3.	Click "Next", choose to install from the internet, and click "Next" twice to accept the root install and local package directories. Click "Next" to accept your internet connection settings.
+4.	Select any package download site from the list. Click "Next" to download the packages from the mirror download site.
+5.	At this screen, you can search and select packages to install. I recommend `curl`, `wget`, `openssh`, `git`, and `vim` to get started. You can expand package categories using the `+` to the left of the category name. ![Package List](https://raw.githubusercontent.com/jay-hankins/CSToolsCourse/master/tutorials/shells/cygwin_screenshots/Packages.PNG)
 
+6.	Mark a package to be downloaded by clicking the `⟳` to the left of "Skip". ![Marked package to install](https://github.com/jay-hankins/CSToolsCourse/raw/master/tutorials/shells/cygwin_screenshots/search_openssh.PNG)
 
-6. Mark a package to be downloaded by clicking the `⟳` to the left of "Skip".
-![Marked package to install](https://github.com/jay-hankins/CSToolsCourse/raw/master/tutorials/shells/cygwin_screenshots/search_openssh.PNG)
+7.	Click "Next" when you've selected your desired packages, and "Next" again to download and install them. Wait patiently while the Cygwin installer downloads and installs the packages and their dependencies.
 
+8.	Click "Finish" to complete the installation.
 
-7. Click "Next" when you've selected your desired packages, and "Next" again to download and install them. Wait patiently while the Cygwin installer downloads and installs the packages and their dependencies.
-8. Click "Finish" to complete the installation.
+How do I use it?
```

Comment:
I set up my editor to do it this way because that's my preference but I can change it back.